packs/spells/qi-blast.json

```
@@ -122,7 +122,64 @@
             "value": ""
         },
         "requirements": "",
-        "rules": [],
+        "rules": [
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "label": "PF2E.SpecificRule.Monk.QiSpells.HeavensThunder.RollOptionLabel",
+                "option": "heavens-thunder",
+                "placement": "spellcasting",
+                "predicate": [
+                    "self:effect:heavens-thunder"
+                ],
+                "suboptions": [
+                    {
+                        "label": "PF2E.TraitElectricity",
+                        "value": "electricity"
+                    },
+                    {
+                        "label": "PF2E.TraitSonic",
+                        "value": "sonic"
+                    }
+                ],
+                "toggleable": true
+            },
+            {
+                "key": "DamageAlteration",
+                "mode": "override",
+                "predicate": [
+                    "heavens-thunder",
+                    "item:slug:qi-blast"
+                ],
+                "property": "damage-type",
+                "selectors": [
+                    "spell-damage"
+                ],
```

```suggestion
                "predicate": [
                    "heavens-thunder"
                ],
                "property": "damage-type",
                "selectors": [
                    "{item|id}-damage"
                ],
```
If the rule is going to be on the spell, then this can be trimmed down